.github/workflows/sbt.yml

```diff
@@ -1,48 +1,29 @@
 name: Scala CI
 
-# Support "official releases" on master, and release fixes on release/ prefixed branches
 on:
   push:
     branches:
       - 'master'
-      - 'releases/**'
-      - '!releases/**-alpha'
```

Comment:
These didn't seem to be used, plus you'll get SNAPSHOTS being published now. This just simplifies things.